docs/self-hosted-getting-started.md

```diff
@@ -0,0 +1,99 @@
+# Quick Start: Run a self-hosted database instance with automatic vectorization
+
+If you want to see how it works, follow these steps to setup 
+a self-hosted database instance with automatic embedding vectorization:
+
+## Pre-requisites
+- Docker installed on your machine
+- An OpenAI API key. You can create one [here](https://platform.openai.com/api-keys).
+
+## Setup
+
+1. Run the following docker-compose file to start a TimescaleDB instance and the Vectorizer worker: 
+    ```yaml
+    services:
+      db:
```

Comment:
Add a `healthcheck` like in https://github.com/timescale/pgai/pull/113/files#diff-facaded51b7d1c9656ae43b449b69aa398fee9129321a0001d97048c00c8cc1cR9-R15 so that the other container starts when this one is ready to accept connections. Otherwise, you might get error while postgres is starting.

Reply:
Since we now start the containers 1 by 1 I don't think we need this. Just bloats the example and makes a user have to understand a more complex compose. I also removed the `depends_on`.